Test/QuickCheck/Arbitrary.hs

```diff
@@ -579,6 +579,7 @@ instance Arbitrary Word64 where
   shrink    = shrinkIntegral
 
 instance Arbitrary Char where
+  -- | This currently only generates Latin1 Chars, if you want Unicode use the @Unicode@ newtype
```

Comment:
It probably should be above the instance. Could you generate docs locally to check?